tools/omicron-install-dendrite-mac.sh

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env bash
```

Comment:
would probably make sense to change the name since it installs both dendrite and maghemite

Reply:
yep. was just doing this haha.